server/src/domain/user/response-dto/user-response.dto.ts

```diff
@@ -1,10 +1,17 @@
-import { UserEntity } from '@app/infra/entities';
+import { UserAvatarColor, UserEntity } from '@app/infra/entities';
+
+export const getRandomAvatarColor = (): UserAvatarColor => {
+  const values = Object.values(UserAvatarColor);
+  const randomIndex = Math.floor(Math.random() * values.length);
+  return values[randomIndex] as UserAvatarColor;
+};
```

Comment:
Why don't we just make this deterministic? I don't think need need to store it or anything like that. Empty implies random and if it was deterministic this could be accomplished in the mapper itself.

Reply:
I don't follow. The goal is to create a random avatar color only on user creation.